src/openms/include/OpenMS/FORMAT/OMSFileLoad.h

```diff
@@ -29,20 +29,19 @@
 //
 // --------------------------------------------------------------------------
 // $Maintainer: Hendrik Weisser $
-// $Authors: Hendrik Weisser $
+// $Authors: Hendrik Weisser, Chris Bielow $
 // --------------------------------------------------------------------------
 
 #pragma once
 
 #include <OpenMS/CONCEPT/ProgressLogger.h>
 #include <OpenMS/KERNEL/FeatureMap.h>
 #include <OpenMS/METADATA/ID/IdentificationData.h>
+#include <OpenMS/FORMAT/OMSFileStore.h>
 
-#include <QtCore/QJsonArray> // for JSON export
-#include <QString>
-#include <QStringList> // avoid compiler error on initialization using init. list
+#include <SQLiteCpp/Database.h>
```

Comment:
could this be replaced by forward declarations?

Reply:
Was about to ask, then it could stay a private dependency.